packages/utils/src/lib/report.unit.test.ts

```diff
@@ -55,11 +55,12 @@ describe('countWeightedRefs', () => {
 
 describe('compareIssueSeverity', () => {
   it('should order severities in logically ascending order when used as compareFn with .sort()', () => {
-    expect(
-      (['error', 'info', 'warning'] satisfies IssueSeverity[]).sort(
-        compareIssueSeverity,
-      ),
-    ).toEqual(['info', 'warning', 'error']);
+    const issueArr = ['error', 'info', 'warning'] satisfies IssueSeverity[];
+    expect([...issueArr].sort(compareIssueSeverity)).toEqual([
```

Comment:
I would just add an ESLint disable comment here instead, the cloning is unnecessary here, the array isn't used anywhere else.

Reply:
Alright, reverting this change - it can only be disabled once the rule itself is enabled and there are still other errors in this package.